sdk/keyvault/azure-security-keyvault/azure/security/keyvault/aio/__init__.py

```diff
@@ -3,7 +3,6 @@
 # Licensed under the MIT License. See LICENSE.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-from .secrets._client import SecretClient
-from ..secrets._models import Secret, SecretAttributes, DeletedSecret
+from .vault_client import VaultClient
 
-__all__ = ["SecretClient", "SecretAttributes", "Secret", "DeletedSecret"]
+__all__ = ["VaultClient"]
```

Comment:
This should include `KeyClient` and `SecretClient` as well so users can do e.g. `from azure.security.keyvault.aio import KeyClient`